### PR TITLE
RE2022-240: Add specs for genome attribs columns & loader

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: src/common/collection_column_specs/genome_attribs-GROW.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+genome_attribs-PMI.yml
```
Comment (MrCreosote) [inline src/common/collection_column_specs/genome_attribs-GROW.yml:1]: softlink

File: src/common/collection_column_specs/genome_attribs-GTDB.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   - key: checkm_completeness
+     type: float
+
+   - key: checkm_contamination
+     type: float
+
+   - key: checkm_marker_count
+     type: int
+
+   - key: checkm_marker_lineage  # GTDB node, might need better tokentizing or substring search
```
Comment (Tianhao-Gu) [inline src/common/collection_column_specs/genome_attribs-GTDB.yml:19]: * tokenizing

**Review by Tianhao-Gu — COMMENTED** [review]
spec and code changes look good to me. Do you think it's a good idea to add a note for `SELECTED_FEATURES` to remind us if we make any changes to selected features should prompt corresponding modifications to the spec file?

**Comment by MrCreosote** [comment]
> Do you think it's a good idea to add a note for SELECTED_FEATURES to remind us if we make any changes to selected features should prompt corresponding modifications to the spec file?

Maybe it should be the other way around and the specs determine the selected features, possibly with a mapping from the original name to a new name? And If the output is missing a key an error is thrown?

File: src/common/collection_column_specs/genome_attribs-GTDB.yml (hunk 2 above)
Comment (MrCreosote) [reply src/common/collection_column_specs/genome_attribs-GTDB.yml]: fixed

**Comment by Tianhao-Gu** [comment]
> > Do you think it's a good idea to add a note for SELECTED_FEATURES to remind us if we make any changes to selected features should prompt corresponding modifications to the spec file?
> 
> Maybe it should be the other way around and the specs determine the selected features, possibly with a mapping from the original name to a new name? And If the output is missing a key an error is thrown?

specs determines SELECTED_FEATURES makes more sense to me. But this PR LGTM.

**Review by Tianhao-Gu — APPROVED** [review]

**Pull request merged**